server/src/schemas/currencyLedgerSchema.ts

```diff
@@ -0,0 +1,18 @@
+import { gql } from "graphql-tag";
+
+export const CurrencyLedgerTypeDefs = gql`
+  type CurrencyLedgerEntry {
+    id: ID!
+    dateTime: DateTime!
+    accountID: ID!
+    amount: Int!
+    source: String!
+    description: String
+    atxID: Int
+    refID: Int
+  }
+
+  extend type Query {
+    currencyLedgerEntriesLimit(searchText: String): [CurrencyLedgerEntry]
```

Comment:
should you call it currencyLedgerLimit100 or maybe currencyLedgerLimit(searchText: string, limit: number)